src/views/CronTabList/CronTabList.tsx

```diff
@@ -54,77 +80,178 @@ const CronTabList: React.FC<CronTabListProps> = ({ namespace }) => {
   );
 };
 
-const cronTabListRow: React.FC<RowProps<CronTabKind>> = ({ obj, activeColumnIDs }) => {
+const CronTabKebab: React.FC<CronTabKebabProps> = ({ obj }) => {
+  const { t } = useCronTabTranslation();
+  const [isOpen, setIsOpen] = React.useState(false);
+  const launchDeleteModal = useDeleteModal(obj);
+  const { name, namespace } = obj.metadata;
+  const canEditCronTab = useAccessReview({
+    group: CronTabModel.apiGroup,
+    resource: CronTabModel.labelPlural,
+    verb: "update",
+    name,
+    namespace,
+  });
+  const canDeleteCronTab = useAccessReview({
+    group: CronTabModel.apiGroup,
+    resource: CronTabModel.labelPlural,
+    verb: "delete",
+    name,
+    namespace,
+  });
+  const history = useHistory();
+
+  const onToggle = (isOpen: boolean) => {
+    setIsOpen(isOpen);
+  };
+
+  const onFocus = () => {
+    const element = document.getElementById("kebab-button");
+    element.focus();
+  };
+
+  const onSelect = () => {
+    setIsOpen(false);
+    onFocus();
+  };
+
+  const editURL = `/k8s/ns/${namespace}/${cronTabGroupVersionKind.group}~${
+    cronTabGroupVersionKind.version
+  }~${cronTabGroupVersionKind.kind}/${encodeURIComponent(name)}/yaml`;
+  const editLabel = "Edit CronTab";
```

Comment:
```suggestion
  const editLabel = t("Edit CronTab");
```

Reply:
Good catch.  This should be an untranslated string (as should `deleteLabel` below) since they are used for `data-test-action` as well as the item label (which wrap them with `t()`.  Will correct `deleteLabel` to match.